TeknoParrotUi.Common/Descriptions/HOTD4.json

```diff
@@ -7,5 +7,5 @@
   "amd_issues": null,
   "intel": "NO_INFO",
   "intel_issues": null,
-  "general_issues": null
+  "general_issues": "Test menu needs a fix to run"
```

Comment:
not an issue, game requires extra files.

Reply:
OK, re-issued.